modules/cli/src/main/java/org/apache/ignite/cli/core/exception/handler/SqlExceptionHandler.java

```diff
@@ -39,8 +48,48 @@ public class SqlExceptionHandler implements ExceptionHandler<SQLException> {
 
     public static final String CONNECTION_BROKE_MESSAGE = "Connection error";
 
+    private final Map<Integer, Function<IgniteException, ErrorComponentBuilder>> sqlExceptionMappers = new HashMap<>();
+
+    private final Function<IgniteException, ErrorComponentBuilder> defaultErrorMapper =
+            (e) -> ErrorUiComponent.builder().header("Unrecognized error while process SQL query" + e.getMessage());
```

Comment:
Maybe use a method similar to invalidQueryErrUiComponent to extract code, traceId and cause message?
Also "process"->"processing"